apps/system/utils/fscmd.c

```diff
@@ -1097,15 +1113,13 @@ static int tash_rm(int argc, char **args)
 		ret = unlink(fullpath);
 		if (ret != OK) {
 			FSCMD_OUTPUT(CMD_FAILED, args[0], "unlink");
-			fscmd_free(fullpath);
-			return ret;
+			goto errout;
```

Comment:
`unlink` also sets the errno on failure case. How about setting the `ret` as -errno?

Reply:
Updated, thanks